docs/current_docs/api/snippets/constructors/python/complex/main.py

```diff
@@ -0,0 +1,40 @@
+from typing import Annotated
+import dagger
+from dagger import (
+    dag,
+    DefaultPath,
+    Doc,
+    field,
+    object_type,
+)
+
+
+
+
```

Comment:
Why so much empty lines? Normally only 2 should be good no?

Reply:
Thanks for catching this!